code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -342,3 +341,64 @@
 	. = ..()
 	reagents.add_reagent("lipozine", 2)
 	color = reagents.get_color()
+
+//// Recreational drugs for loadout
+/obj/item/reagent_containers/pill/bliss
+	name = "mysterious pill"
+	desc = "You're not quite sure what's in this..."
+	icon_state = "pill1"
+
+/obj/item/reagent_containers/pill/bliss/Initialize()
+	. = ..()
+	reagents.add_reagent("bliss", 10)
+	color = reagents.get_color()

```

Comment:
You could subtype pill to unidentified, then subtype that to each individual drug, such that you only have to define this once, and then each individual drug defines the initialize() call to add that drug specifically.
E.g.,
```suggestion
/obj/item/reagent_containers/pill/unidentified
	name = "mysterious pill"
	desc = "You're not quite sure what's in this..."
	icon_state = "pill1"


/obj/item/reagent_containers/pill/unidentified/bliss/Initialize()
	. = ..()
	reagents.add_reagent("bliss", 10)
	color = reagents.get_color()
```
(And repeat _just_ the initialize definitions for all the other drugs you're adding)